frontend/src/app/core/file-upload/op-file-upload.service.ts

```diff
@@ -98,6 +105,7 @@ export class OpenProjectFileUploadService {
    * @param {string} url
    * @param {UploadFile} file
    * @param {string} method
+   * @param {'text'|'json'} responseType
```

Comment:
I would suggest, adding the return type to the function and remove the whole comment.

Reply:
fixed